cmd/containerd/builtins_overlay_linux.go

```diff
@@ -0,0 +1,19 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import _ "github.com/containerd/containerd/snapshots/overlay/plugin"
```

Comment:
We already have this in https://github.com/containerd/containerd/blob/5da21f1ded9700797bd7ba43759c3948afdcca8e/cmd/containerd/builtins_linux.go#L26

Reply:
Thanks

I am using 1.4.4.😓
 👌I will try 1.5.0 later.